workers/unity/Packages/com.improbable.gdk.tools/Plugins/PluginPostprocessor.cs

```diff
@@ -0,0 +1,133 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+
+namespace Improbable.Gdk.Tools
+{
+    internal class PluginPostprocessor : AssetPostprocessor
+    {
+        private static readonly List<PluginCompatibilitySetting> PluginCompatibilitySettings = new List<PluginCompatibilitySetting>
+        {
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.StandaloneOSX
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.StandaloneLinux64
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.StandaloneWindows64,
+                CPUType.X86_64
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.StandaloneWindows,
+                CPUType.X86
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.Android,
+                CPUType.Arm64
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.Android,
+                CPUType.ARMv7
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.Android,
+                CPUType.X86
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Core,
+                BuildTarget.iOS
+                ),
+            new PluginCompatibilitySetting(
+                PluginType.Sdk,
+                new List<BuildTarget> { BuildTarget.iOS }
```

Comment:
Any reason this is a `List`?

Reply:
just in case we need to add more platforms in the feature and to differentiate between compatible and incompatible platforms.
could also be an array or I use static methods to initalize it. What would you prefer?

Reply:
This is fine, I was just curious because it stood out from the other entries.